src/Database/Traits/Revisionable.php

```diff
@@ -157,7 +157,8 @@ protected function revisionableCleanUp()
 
     protected function revisionableGetCastType($attribute)
     {
-        if (in_array($attribute, $this->getDates())) {
+        // dates property deprecated in Laravel 10, no longer used in getDates()
+        if (in_array($attribute, $this->getDates()) || $this->isDateCastable($attribute) || in_array($attribute, $this->dates)) {
```

Comment:
Could getDates() return the value of $this->dates and then could we change isDateCastable() to look at those properties? Or just have something that automatically moves the values from $dates to $casts?